py3status/modules/hueshift.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+"""
+Shift color temperature on the screen.
+
+Configuration parameters:
+    button_down: mouse button to change color temperature (default 5)
+    button_reset: mouse button to reset color temperature (default 3)
+    button_up: mouse button to change color temperature (default 4)
+    command: specify blueshift, redshift, or sct to use, otherwise automatic
+        (default None)
+    delta: specify interval value to change color temperature (default 250)
+    format: display format for this module
+        *(default '\?if=pgrep&color=bad \u263c!'
+        '|\?color=hue \u263c {temperature}K')*
+    initial: specify initial color temperature to use (default 6500)
+    maximum: specify maximum color temperature to use (default 25000)
+    minimum: specify minimum color temperature to use (default 1000)
+    thresholds: specify color thresholds to use
+        (default [(6499, '#f3c'), (6500, '#ff3'), (6501, '#3cf')])
+
+Control placeholder:
+    {pgrep} a boolean based on pgrep processing data
+
+Format placeholder:
+    {temperature} color temperature, eg 6500
+    {command} command name, eg blueshift, redshift, sct
+
+Color options:
+    color_bad: disabled due to running processes
+
+Color thresholds:
+    hue: print a color based on the value of color temperature
+
+Requires:
+    blueshift: an extensible and highly configurable alternative
+    redshift: program to adjust the color temperature of your screen
+    sct: set color temperature
+
+Suggestions:
+    campfire: 4500
+    dust storm on mars: 2000
+    coffee free all nighter: 8000
+
+Examples:
+```
+# good rules; show command name; don't colorize everything
+hueshift {
+    format = '[\?if=pgrep&color=bad \u263c!|[\?color=hue&show \u263c '
```

Comment:
this needs `u'...'` for python2 as has unicode

I'd also prefer a nice error for 'color manager is running so we cannot' type thing rather than complex format